PusherServer/BatchEvent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PusherServer
+{
+    /// <summary>
+    /// Represents an event that is part of a batch trigger
+    /// </summary>
+    public class BatchEvent
+    {
+        /// <summary>
+        /// The name of the event
+        /// </summary>
+        public string name { get; set; }
+
+        /// <summary>
+        /// The event data
+        /// </summary>
+        public string data { get; set; }
```

Comment:
I think a `SerializeAs` attribute may be supported. See: https://github.com/restsharp/RestSharp/blob/658f0984e3ec71f2119f50f7de878f155246e9ce/RestSharp.Tests/SerializerTests.cs#L261

We'll need to test.